src/common/hooks/auth/use-save-auth-request-callback.ts

```diff
@@ -1,19 +1,20 @@
 import { useCallback, useEffect } from 'react';
 import { useLocation } from 'react-router-dom';
-import { useSetRecoilState } from 'recoil';
+
 import { decodeToken } from 'jsontokens';
 import { DecodedAuthRequest } from '@common/dev/types';
 import { useWallet } from '@common/hooks/use-wallet';
 import { getRequestOrigin, StorageKey } from '@common/storage';
 import { ScreenPaths } from '@store/common/types';
 import { useOnboardingState } from './use-onboarding-state';
 import { authRequestState, currentScreenState } from '@store/onboarding';
+import { useUpdateAtom } from 'jotai/utils';
 
 export function useSaveAuthRequest() {
   const { wallet } = useWallet();
   const { screen } = useOnboardingState();
-  const changeScreen = useSetRecoilState(currentScreenState);
-  const saveAuthRequest = useSetRecoilState(authRequestState);
+  const changeScreen = useUpdateAtom(currentScreenState);
+  const saveAuthRequest = useUpdateAtom(authRequestState);
```

Comment:
Is there any difference b/w using set from `useAtom` and using `useUpdateAtom`? Nothing, right?

Reply:
`useUpdateAtom` exists so you don't need to restructure the response from `useAtom` and have an unused const

Reply:
but they are the same thing, yeah, see https://github.com/pmndrs/jotai/blob/master/src/utils/useUpdateAtom.ts#L7

Reply:
Ok, yep, just wanted to make sure I wasn't missing something. 👍